config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get 'users', to: 'users#index'
+  get 'users/:id', to: 'users#show'
+  get 'users/:id/posts', to: 'posts#index'
+  get 'users/:id/posts/:id', to: 'posts#show'
```

Comment:
- [ ] [OPTIONAL] You can use something like the following to refactor your code to be more readable and easier to update later on.
```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
Thank you for the good review and the good suggestion. I will apply it in the future